contracts/factory/APMRegistryFactory.sol

```diff
@@ -53,41 +53,55 @@ contract APMRegistryFactory is DAOFactory, AppProxyFactory, APMRegistryConstants
         bytes32 namespace = dao.APP_BASES_NAMESPACE();
 
         // Deploy app proxies
-        ENSSubdomainRegistrar ensSub = ENSSubdomainRegistrar(dao.newAppInstance(ENS_SUB_APP_ID, ensSubdomainRegistrarBase));
-        APMRegistry apm = APMRegistry(dao.newAppInstance(APM_APP_ID, registryBase));
+        ENSSubdomainRegistrar ensSub = ENSSubdomainRegistrar(dao.newAppInstance(keccak256(node, keccak256(ENS_SUB_APP_NAME)), ensSubdomainRegistrarBase));
+        APMRegistry apm = APMRegistry(dao.newAppInstance(keccak256(node, keccak256(APM_APP_NAME)), registryBase));
```

Comment:
From what I can tell, these changes are to make the `ENSSubdomainRegistrar`, `APMRegistry`, and `Repo` appIds match their namehash values. Is this mostly cosmetic or is the plan to use this with ENS in some way?

Reply:
It allows node-aragon to recognize APM apps when traversing an APM org